canonical/rebac-admin
# feat: add group panel

## Done

- Add the add group panel.

## QA

- Go to the groups page and click 'Create group'.
- Fill in the details and submit.
- The panel should close and the list should fresh (without the new group as it's using the mock API).

## Details

- https://warthogs.atlassian.net/browse/WD-12384
